libc/src/stdlib/CMakeLists.txt

```diff
@@ -497,15 +497,24 @@ add_entrypoint_object(
     .exit_handler
 )
 
+list(APPEND exit_deps
+  libc.src.__support.OSUtil.osutil
+  ._Exit
+)
+if (NOT LIBC_TARGET_OS_IS_BAREMETAL)
+  # list(APPEND exit_deps
```

Comment:
why are these commented out?

Reply:
I forgot to save the file before committing, whoops :P